From e04802d7ce5b51575fcf8ad4dd18177fe079f7aa Mon Sep 17 00:00:00 2001
From: Michael Pyne <mpyne@kde.org>
Date: Thu, 31 Dec 2015 12:32:01 -0500
Subject: [PATCH 4/4] Fix build on Gentoo with recent GStreamer 1.0.

Partially fixed by RR 125690, this also fixes for Gentoo in my testing,
and should for other distros where gstreamer places its gst-config under
gstreamer-1.0/include.

REVIEW:126554
---
 cmake/FindGStreamer.cmake | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/cmake/FindGStreamer.cmake b/cmake/FindGStreamer.cmake
index e30c349..1375b30 100644
--- a/cmake/FindGStreamer.cmake
+++ b/cmake/FindGStreamer.cmake
@@ -44,7 +44,7 @@ ENDIF (NOT GSTREAMER_INCLUDE_DIR)
 FIND_PATH(GSTREAMER_CONF_INCLUDE_DIR gst/gstconfig.h
    PATHS
    ${PKG_GSTREAMER_INCLUDE_DIRS}
-   PATH_SUFFIXES gstreamer-1.0
+   PATH_SUFFIXES gstreamer-1.0 gstreamer-1.0/include
    )
 
 IF (NOT GSTREAMER_CONF_INCLUDE_DIR)
-- 
1.9.5

